fdbserver/ptxn/TLogServer.actor.cpp

```diff
@@ -1527,14 +1524,18 @@ ACTOR Future<Void> startTLogServers(std::vector<Future<Void>> actors,
 
 TEST_CASE("/fdbserver/ptxn/test/run_tlog_server") {
 	test::TestDriverOptions options(params);
+	// Commit validation in real TLog is not supported for now
+	options.skipCommitValidation = true;
```

Comment:
override skipCommitValidation = true for some random unit tests that set it false.